ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java

```diff
@@ -216,20 +213,4 @@ private void handleResponseEnd() {
 
   }
 
-  private CompletableFuture<InsertsStreamSubscriber> createInsertsSubscriberAsync(
-      final String target,
-      final JsonObject properties, final Subscriber<InsertResult> acksSubscriber,
-      final Context context,
-      final ApiSecurityContext apiSecurityContext) {
-    final VertxCompletableFuture<InsertsStreamSubscriber> vcf = new VertxCompletableFuture<>();
-    workerExecutor.executeBlocking(
-        p -> p.complete(
-            endpoints.createInsertsSubscriber(target, properties, acksSubscriber, context,
-                workerExecutor, apiSecurityContext)),
-        false,
```

Comment:
As part of moving the async handling for creating `InsertsStreamSubsriber`s to `KsqlServerEndpoints`, this optimization for non-serial blocking code execution was lost. I assume that's no big deal, but wanted to call it out in case the optimization was significant.

Reply:
Good catch! That's a bug.